Agreements/Internal/TCNE_TCRED.txt

```diff
@@ -0,0 +1,5 @@
+COPX:EGSS:*:*:CLN:*:London TC NE:London TC REDFA:11000:*:^CLN
```

Comment:
Does this differ for aircraft routing east vs. south east after CLN? Might need to be a restriction at EVMEK to get aircraft over the top of the JACKO sector?

Reply:
No, all go to 110 to TC REDFA, 

REDFA then climbs eastbounds to 210 and hands to AC Clacton
or 
climbs southbounds to 150 and hands to TC SABER